andela/ah-jumanji-frontend
# #163518809:Add error handing to the application

#### What does this PR do?
It adds error handling to the application to show custom messages when errors happen

#### Description of Task to be completed?
define error message components for different errors and add an error boundary to handle all other errors.

#### How should this be manually tested?
Browse to a non-existent URL, delete the auth token on the application or raise an exception within any of the components in the application and you should be presented with an error page

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[163518809](https://www.pivotaltracker.com/story/show/163518809)

#### Screenshots (if appropriate)
<img width="1276" alt="image" src="https://user-images.githubusercontent.com/20369585/52077146-5656c400-25a1-11e9-93e9-0a6433ed6490.png">

<img width="1276" alt="image" src="https://user-images.githubusercontent.com/20369585/52077318-c36a5980-25a1-11e9-951a-56aa7e62da60.png">

#### Checklist:
